--- CuraEngine-7b2b9630c7b5ec5e4534de94b46bb37004cccb88.orig/Makefile	2015-03-03 11:37:04.000000000 +0100
+++ CuraEngine-7b2b9630c7b5ec5e4534de94b46bb37004cccb88/Makefile	2015-04-10 01:00:12.055685766 +0200
@@ -47,10 +47,10 @@
 	ifeq ($(UNAME), Linux)
 		OPEN_HTML=firefox
 		ifeq ($(BUILD_TYPE),DEBUG)
-			LDFLAGS += --static
+#			LDFLAGS += --static
 		else
 			CFLAGS += -flto
-			LDFLAGS += --static -flto
+			LDFLAGS += -flto
 		endif
 	endif
 	ifeq ($(UNAME), OpenBSD)
@@ -81,6 +81,9 @@
 test: $(EXECUTABLE)
 	python tests/runtest.py $(abspath $(EXECUTABLE))
 
+install: $(EXECUTABLE)
+	install -D -m 755 $(EXECUTABLE) $(DESTDIR)/usr/bin/CuraEngine
+
 ## clean stuff
 clean:
 	rm -f $(EXECUTABLE) $(OBJECTS) $(BUILD_DIR)/libclipper.a
